main.js

```diff
@@ -6,7 +6,13 @@
  * @typedef {import("./typings/metrics").TickingMetric} TickingMetric
  */
 
-require('isomorphic-fetch');
+// only include isomorphic-fetch if the application has explicitly disabled native fetch
```

Comment:
**suggestion**: could we expand this comment to add a bit of context on why we're doing this? afaict it's to satisfy problem hub as there won't be any behavioural change (as `isomorphic-fetch` already checks if native fetch is present)

Reply:
so this idea is [from Jen's original native fetch investigation](https://financialtimes.atlassian.net/wiki/spaces/CPP/pages/8042020974/Node.js+18+and+Native+Fetch#Moving-to-future-state---one-option). which i should have read a bit closer, because the reason she's suggesting doing it this way is so we can add a warning here. i'll do that.